pipelines/tekton/aiedge-e2e/test-data/bike-rentals-test-data-cm.yaml

```
@@ -2,6 +2,8 @@ kind: ConfigMap
 apiVersion: v1
 metadata:
   name: bike-rentals-test-data
+  labels:
+    model-name: bike-rentals-test-data
```

```suggestion
    model-name: bike-rentals-auto-ml
```

^this is the model name 🙂